packages/website/pages/docs/reference/node-troubleshooting.mdx

```diff
@@ -10,6 +10,12 @@ You can ignore any WARN logs.
 
 Make sure that your RPC is pointing to an Sepolia archive node, and not a full node.
 
+#### `error="initialize rpc clients error"`
+
+The RPC in your config is wrong. You have to set the ENDPOINT correctly. Please refer to the following page to set the correct RPC https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node
```

Comment:
```suggestion
The RPC in your config is incorrect. You have to set the ENDPOINT correctly.  Please refer to the following page to set the correct RPC https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node.
```

Reply:
Thank you for confirmation
Wrong doesn't appear and incorrect is the correct sentence (I'm not good at English, so I made a mistake, sorry)